packages/Textarea/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@paprika/textarea",
+  "version": "0.1.0",
+  "description": "Textarea component",
+  "author": "@paprika",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/textarea"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/stylers": "^0.1.0",
+    "@paprika/tokens": "^0.1.0",
+    "class-names": "^1.0.0",
```

Comment:
Should be `classnames`.

Reply:
Oh i'll fix it next PR

Reply:
We should turn `import/no-extraneous-dependencies` back on